src/spectrecoff/Prompt.fs

```diff
@@ -17,10 +22,11 @@ module private Prompts =
         prompt.Title <- question
         prompt
 
-    let multiSelectionPrompt question choices = 
+    let multiSelectionPrompt question choices (options: MultiSelectionPromptOptions) = 
```

Comment:
Now that there is one more argument, maybe order them in same order as in the calling location

`let multiSelectionPrompt options choices question = ...`

Reply:
I used the order that all other functions around it use - and there it's also "mixed up"